integration-test/src/test/java/org/ehcache/integration/ThreadPoolsTest.java

```diff
@@ -102,4 +106,23 @@ public void testDefaultThreadPoolFailsWithExplicitConfig() throws Exception {
     cacheManager.close();
   }
 
+  @Test
+  public void testNoDefaultThreadPoolFailsWithNoCacheEventDispatcher() throws Exception {
```

Comment:
This test could stay and be adjusted to the new message, but it probably makes more sense to remove it. It could become a test of the PooledExecutorService though.

Reply:
Removed